package.json

```diff
@@ -23,7 +23,7 @@
     "path-browserify": "^1.0.1",
     "react": "^18.2.0",
     "react-redux": "^8.1.0",
-    "terraso-backend": "github:techmatters/terraso-backend#7d5820d",
+    "terraso-backend": "github:techmatters/terraso-backend#8ac6f4e",
```

Comment:
request: @josebui Could you also run `npm install` to regenerate the lock file?

Reply:
@david-code done